src/components/common/header/Header.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import AuthOptions from '../auth/AuthOptions';
-import pear from './pear.png';
+import AuthOptions from '../../auth/AuthOptions';
+import pear from '../../../pear.png';
+import { useSelector } from 'react-redux';
 
 export default function Header() {
-  return (
+  const user = useSelector((state) => state.userReducer.user);
+  return !!user ? (
```

Comment:
Made header only show when logged in, purely UI decision to make pre-login more flashy